### DISCUSSION TITLE
Accessing Django admin

### DISCUSSION THREAD

**Discussion by JWingate80** [amidaware/tacticalrmm#1368] [General]
When trying to log into the Django administration area I get the following message: Please enter the correct username and password for a staff account. Note that both fields may be case-sensitive.

Instructions were followed from the documentation found here: https://docs.tacticalrmm.com/functions/django_admin/

**Comment by dinger1986** [comment]
can you please share screenshots

**Comment by JWingate80** [comment]
> can you please share screenshots

![image](https://user-images.githubusercontent.com/10959760/206265579-a38b9200-212e-4090-969d-ea9d9835f753.png)

**Comment by silversword411** [comment]
Where are you getting your username from? 

You're probably using the wrong ID/pass.

**Reply by JWingate80** [reply]
> Where are you getting your username from?
> 
> You're probably using the wrong ID/pass.

I'm using the same login I use to log into the Tactical RMM web UI as it says to do in the documentation I linked to above. 
![image](https://user-images.githubusercontent.com/10959760/206496190-2f9c8b3e-957d-4f83-9811-f14c482d7bb6.png)

I am highly confident this isn't the info being entered incorrectly.

**Reply by wh1te909** [reply]
probably not a django superuser, only those accounts can login to django admin
try with the account you created in the installs cript

**Comment by silversword411** [comment]
https://github.com/amidaware/trmm-docs/commit/239ebc1b4fba03f7147716acd4d90e373bb4215a
